src/Extension/Settings/settings.yaml

```diff
@@ -46,7 +46,7 @@ settings:
     init: true
     is_project: false
     type: path
-    values: './'
+    values: '/data'
```

Comment:
revert

Reply:
This is a default folder for sdk in the image